images/linux/Ubuntu2004-Readme.md

```diff
@@ -103,30 +101,30 @@
 - Skopeo 1.5.0 (apt source repository: https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable)
 - Sphinx Open Source Search Server 2.2.11
 - SVN 1.13.0
-- Terraform 1.2.6
+- Terraform 1.2.7
 - yamllint 1.27.1
 - yq 4.27.2
 - zstd 1.5.2 (homebrew)
 
 ### CLI Tools
 - Alibaba Cloud CLI 3.0.124
-- AWS CLI 2.7.21
+- AWS CLI 2.7.22
 - AWS CLI Session manager plugin 1.2.339.0
 - AWS SAM CLI 1.53.0
 - Azure CLI (azure-cli) 2.39.0 (installation method: https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-linux?pivots=apt)
 - Azure CLI (azure-devops) 0.25.0
-- GitHub CLI 2.14.3
+- GitHub CLI 2.14.4
 - Google Cloud SDK 369.0.0 (apt source repository: https://packages.cloud.google.com/apt)
 - Hub CLI 2.14.2
-- Netlify CLI 10.15.0
-- OpenShift CLI 4.10.25
```

Comment:
looks like broken to me

Reply:
I think we can fix manually the version in the doc:
```
PS > ./oc version
Client Version: 4.11.0
Kustomize Version: v4.5.4
```

Reply:
true that, will fix a swreport function